### PR TITLE
Rename Regulator to Sampler

### PR DESCRIPTION
**Pull request by hasit**
### Description of change

Screenshot 2023-06-12 at 10 39 32 AM

##### Checklist

- [x] Tested in playground or other setup
- [x] Screenshot (Grafana) from playground added to PR for 15+ minute run
- [x] Documentation is changed or added
- [ ] Tests and/or benchmarks are included
- [ ] Breaking changes

### Summary by OpenAI

**Refactor:**
- Renamed `RegulatorInfo` to `SamplerInfo` in protobuf messages
- Renamed `Regulator` component to `Sampler` throughout the codebase

**Documentation:**
- Updated all references and documentation related to the renaming of the `Regulator` component to `Sampler` in Aperture

> 🎉 Sampler's here, Regulator's gone,
> A new era, a fresh dawn! 🌅
> With clarity we stride ahead,
> Embracing change, no fear or dread. 😄

### CODE REVIEW

File: pkg/otelcollector/consts/consts.go [hunk 1]
```diff
@@ -61,10 +61,10 @@ const (
 	ApertureLoadSchedulersLabel = "aperture.load_schedulers"
 	// ApertureDroppingLoadSchedulersLabel describes load schedulers dropping the traffic.
 	ApertureDroppingLoadSchedulersLabel = "aperture.dropping_load_schedulers"
-	// ApertureLoadRegulatorsLabel describes load regulators matched to the traffic.
-	ApertureLoadRegulatorsLabel = "aperture.load_regulators"
-	// ApertureDroppingLoadRegulatorsLabel describes load regulators dropping the traffic.
-	ApertureDroppingLoadRegulatorsLabel = "aperture.dropping_load_regulators"
+	// ApertureLoadSamplersLabel describes load samplers matched to the traffic.
+	ApertureLoadSamplersLabel = "aperture.load_samplers"
```
Comment (tanveergill) [inline pkg/otelcollector/consts/consts.go:65]: just call it samplers?

Comment (hasit) [reply pkg/otelcollector/consts/consts.go:65]: Just sampler sounds odd.

Comment (hasit) [reply pkg/otelcollector/consts/consts.go:65]: Making the change afterall.

Comment (hasit) [reply pkg/otelcollector/consts/consts.go]: [649485d](https://github.com/fluxninja/aperture/pull/2140/commits/649485d0bc55002b87cbd75b3f43e61b7f4efcff) addresses this.

**Pull request merged**